ompi/mca/mtl/ofi/mtl_ofi.h

```diff
@@ -81,8 +81,7 @@ extern opal_thread_local int ompi_mtl_ofi_per_thread_ctx;
 #define MCA_MTL_OFI_CID_EXCHANGED     0
 
 typedef struct {
-    uint32_t c_index:30;
-    uint32_t c_index_state:2;
+    uint32_t c_index;
 } c_index_vec_t;
```

Comment:
Why do we need this type now that everything is an int ?

Reply:
we don't need it now.  i'll remove that!